### ISSUE TITLE
United States - AIM-54s

### ISSUE THREAD

**Issue by claudejdev** [PygmalionOfCyprus/cmo-db-requests#1250]
### DB Selector

DB3K

### Affected DBID(s)

*

### Summary of Changes

**DataWeapon**
- [x] AIM-54A

ID | Name
--- | ---
3513 |  Phoenix (1974, Mk47 Mod 0 Motor)
444 | AIM-54A Phoenix (1974, Mk60 Mod 0 Motor)
2784 | AIM-54A Phoenix [Fakour-90 Mod] (Iran, 2012, Pure-SARH)

- Length=3.9
- Weight=443.0
- AirRangeMax=72.9
- AirRangeMin=2.0 [^1]

```
SET DataWeapon.Length=3.9,DataWeapon.Weight=443.0,DataWeapon.AirRangeMax=72.9,DataWeapon.AirRangeMin=2.0
WHERE DataWeapon.ID IN (444,2784,3513);
```

- [x] 2784 | AIM-54A Phoenix [Fakour-90 Mod] (Iran, 2012, Pure-SARH)
- Comments=Iran, 2013, Pure-SARH

- [x] 1398 | AIM-54C Phoenix (1989, Mk47 Mod 1 Smokeless Motor)
- Comments = 1985-2004 [^2]
- Length=3.9
- Weight=480.5
- AirRangeMax=99.4
- AirRangeMin=2.0 [^1]

- [x] **DataWeaponCodes**
- 2012 Lock-On After Launch (LOAL) 
Not sure to understand that one correctly.
![104-105 (3)](https://user-images.githubusercontent.com/35732829/170878943-e4eb9246-1628-4632-b4b8-e3942da60512.jpg)

Note : the C variant is said to have entered in service in 1985. A C Plus (sometimes denoted ++, or ECCM/Sealed) entered in service in 1990-ish. ID 1398 already have its sensor set to a Late 80 Gen. I'm not sure which else impactful changes, gameplay wise, did occur on that version.
TLDR; ID 1398 could be the C+ too.

### Sources

- IHS Jane's. 2021. IHS Jane's Weapons Air-launched 2021-2022. Coulsdon, Surrey, UK: IHS Global.
![104-105 (2)](https://user-images.githubusercontent.com/35732829/170878543-ee2a414f-1171-4612-8588-3cdf8fbc22e0.jpg)

![IMG_20220516_143523 (2)](https://user-images.githubusercontent.com/35732829/170870897-1a61f930-89ed-49ff-a373-e7e0aa4790d1.jpg)

[^1]:[Raytheon AIM-54 Phoenix - Directory of U.S. Military Rockets and Missiles](http://www.designation-systems.net/dusrm/m-54.html)
[^2]:[Navy Retires AIM-54 Phoenix Missile - Navy NewsStand](https://web.archive.org/web/20041026202139/http://www.news.navy.mil/search/display.asp?story_id=15422)

**Comment by PygmalionOfCyprus** [comment]
Added 495. Also made changes to the CWDB version.

**Comment by claudejdev** [comment]
> Fixed 495. Also made changes to the CWDB version.

Yeah, I admit I was too lazy to load up the CWDB and check there too.
Would be wonderful if the two could be somewhat merged.

**Comment by PygmalionOfCyprus** [comment]
I agree. Unfortunately it's not so simple, due to non-technical reasons I can't discuss. Maybe one day.
